server_addon/max/client/ayon_max/plugins/load/load_pointcache.py

```diff
@@ -31,13 +31,21 @@ def load(self, context, name=None, namespace=None, data=None):
         file_path = self.filepath_from_context(context)
         file_path = os.path.normpath(file_path)
 
+        rt.frameRate = get_fps_for_current_context()
```

Comment:
Pretty sure this does nothing currently. And if we're fixing this in a different way later this should also not be here - we don't want to enforce the frame rate of the scene to current task on loading/updating something into the scene.